src/components/pages/areas-manage/AreasManage.js

```diff
@@ -160,11 +160,11 @@ class AreasManage extends React.Component {
                     value={this.form.name}
                     placeholder={this.props.intl.formatMessage({ id: 'areas.nameAreaPlaceholder' })}
                     validations={['required']}
-                    disabled={this.props.saving}
+                    disabled={this.props.saving || this.props.loading}
                     />
                 </div>
               </div>
-              <Button className="c-button" disabled={this.props.saving || this.props.editing ? true : false}><FormattedMessage id="forms.save" /></Button>
+              <Button className="c-button" disabled={this.props.saving || this.props.editing ? true : false || this.props.loading}><FormattedMessage id="forms.save" /></Button>
```

Comment:
The ternary could use some parenthesis to avoid problems:
this.props.saving || (this.props.editing ? true : false) || this.props.loading